packages/story-editor/src/components/publishModal/stories/index.js

```diff
@@ -23,8 +23,16 @@ import { useCallback, useState } from '@googleforcreators/react';
 import StoryContext from '../../../app/story/context';
 import { noop } from '../../../utils/noop';
 import { CheckpointContext } from '../../checklist';
+import InspectorContext from '../../inspector/context';
+import { PageAdvancementPanel, SlugPanel } from '../../panels/document';
 import PublishModal from '../publishModal';
 
+const MockDocumentPane = () => (
+  <>
+    <SlugPanel />
+    <PageAdvancementPanel />
```

Comment:
Just grabbing these two because they have the least amount of providers to wrap and provide fake data for :)